index.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="menu.css">
+    <title>Document</title>
+</head>
+  <body>
+    
+    <nav class="toolbar">
+      <div class="logo"><a href="">My Logo</a></div>
+      <div class="menu_main">
+        <input type="checkbox" class="toggler">
+        <div class="humbuger"><div></div></div>
+        <div class="menu">
+          <div class="overlay">
+            <div>
+              <ul>
+                <li><a href="">Portifolio</a></li>
+                <li><a href="">About</a></li>
+                <li><a href="">Context</a></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+
+    </nav>
+    <section class="headline">
+      <div class="container">
+        <h1>I’m Patrick
+            glad to see you!</h1>
+        <p>I’m a software developer! 
+            I can help you build a product ,
+            feature or website Look through 
+            some of my work and experience! 
+            If you like what you see and have 
+            a project you need coded, 
+            don’t hestiate to contact me.</p>
+        <h3>
+            <a href="">LET’S CONNECT</a> 
+        </h3>  
+        <ul class="social-media">
+            <li><a href="twiter"><img src="../Portfolio-finish-mobile-version/img/twiter.svg" alt=""></a></li>
+            <li><a href="linkedin"><img src="../Portfolio-finish-mobile-version/img/Linkedin.svg" alt=""></a></li>
+            <li><a href="medium"><img src="../Portfolio-finish-mobile-version/img/medium.svg" alt=""></a></li>
+            <li><a href="github"><img src="../Portfolio-finish-mobile-version/img/github.svg" alt=""></a></li>
+            <li><a href="chill"><img src="../Portfolio-finish-mobile-version/img/chill.svg" alt=""></a></li>
+        </ul> 
+      </div>
+    </section>
+    <main class="work">
+      <ul>
+      <li class="item tonic">
+        <img src="../Portfolio-finish-mobile-version/img/work1.svg" alt="">
```

Comment:
- Hey, great job adding the images, however, the images are not showing due to how you've added it file path. Please, update this file path to `  <img src="./img/work1.svg" alt="">` to fix the issue with **ALL YOUR IMAGES** not showing.


:point_right: *This file path issue is applicable to all images that are not showing in your webpage*